taichi/transforms/type_check.cpp

```diff
@@ -414,10 +408,6 @@ namespace irpass {
 void typecheck(IRNode *root) {
   analysis::check_fields_registered(root);
   TypeCheck::run(root);
-  //  if (root->is<Block>() && root->as<Block>()->parent == nullptr) {
-  //    fix_block_parents(root); // hot fix
-  //    verify(root);
-  //  }
```

Comment:
Maybe I should start working on #739?

Reply:
Sounds good!

Reply:
(Sorry I didn't realize these comments are left there on purpose...)